app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -123,11 +130,18 @@ public BasicAWSCredentials s3CmsCredentials(BridgeConfig bridgeConfig) {
     }
 
     @Bean(name = "dynamoDbClient")
-    public AmazonDynamoDBClient dynamoDbClient() {
+    @Resource(name = "awsCredentials")
+    public AmazonDynamoDBClient dynamoDbClient(BasicAWSCredentials awsCredentials) {
         int maxRetries = bridgeConfig().getPropertyAsInt("ddb.max.retries");
         ClientConfiguration awsClientConfig = PredefinedClientConfigurations.dynamoDefault()
                 .withMaxErrorRetry(maxRetries);
-        return new AmazonDynamoDBClient(awsCredentials(), awsClientConfig);
+        return new AmazonDynamoDBClient(awsCredentials, awsClientConfig);
+    }
```

Comment:
Adding the credentials to the input parameters is unnecessary, and generally makes it more difficult to chain Spring beans together.